src/ripple/app/misc/NetworkOPs.cpp

```diff
@@ -598,7 +598,7 @@ class NetworkOPsImp final
     bool const m_standalone;
 
     // The number of nodes that we need to consider ourselves connected.
-    std::size_t const m_network_quorum;
+    std::size_t const minPeerCount_;
```

Comment:
Another mix of naming conventions for members: trailing underscore and leading `m`.

Reply:
Agreed; responded in the other comment.